src/test/resources/org/opensearch/agent/tools/register_flow_agent_of_ragtool_with_neural_query_type_request_body.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Test_Agent_For_RagTool",
+  "type": "flow",
+  "description": "this is a test flow agent in flow",
+  "tools": [
+    {
+      "type": "RAGTool",
+      "description": "A description of the tool",
+      "parameters": {
+        "embedding_model_id": "<MODEL_ID>",
+        "inference_model_id": "c8NwFo0BY4jgIz2mgLNH",
+        "index": "<INDEX_NAME>",
+        "embedding_field": "embedding",
+        "query_type": "neural",
+        "enable_Content_Generation":"false",
```

Comment:
According to our naming convention, this field name should be `enable_content_generation `

Reply:
renamed